python/packages/aws-lambda-sdk/CHANGELOG.md

```diff
@@ -2,6 +2,16 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
+## 0.1.12 (2023-04-05)
+
+### Bug Fixes
+
+- Fix customer module initialization
```

Comment:
Let's also add here `Revert breaking rename of `serverless_sdk` into `sls_sdk` (it belongs to bug fixes as it was a breaking change that was published with a patch bump)

Reply:
Done 👍